src/GitHub.VisualStudio/UI/Views/GitHubConnectContent.xaml.cs

```diff
@@ -101,7 +101,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
 
             var item = values[0] as ISimpleRepositoryModel;
             var context = values[1] as IGitAwareItem;
-            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath == item?.LocalPath;
+            return context?.ActiveRepoUri == item?.CloneUrl && context?.ActiveRepo?.LocalPath.ToUpper() == item?.LocalPath.ToUpper();
```

Comment:
`ToUpper` creates a new string. In this case, two throwaway strings just to compare equality. Also, you can run into the [Turkish I problem here](http://haacked.com/archive/2012/07/05/turkish-i-problem-and-why-you-should-care.aspx/) because no culture was specified.

The safe way to compare strings in a case insensitive manner is to use [`String.Equals`](https://msdn.microsoft.com/en-us/library/t4411bks(v=vs.110).aspx) and pass in a `StringComparison`. For example,

```csharp
String.Equals(context?.ActiveRepo?.LocalPath, item?.LocalPath, StringComparison.OrdinalIgnoreCase);
```